src/components/layout/Header.js

```diff
@@ -1,23 +1,22 @@
 import React, { useEffect, useState } from "react";
+import CurrencyExchange from "../../shared/services/currencyExchange";
+
 
 export default function Header() {
   const [data, setData] = useState([]);
 
   useEffect(() => {
-    fetch(`https://api.privatbank.ua/p24api/pubinfo?json&exchange&coursid=5`)
-      .then((response) => response.json())
-      .then((json) => setData(json));
-  }, []);
-const num = 5.5698768;
-console.log(num.toFixed(2));
-
+    CurrencyExchange()
```

Comment:
1. Can we write CurrencyExchange.then(setData) ?
2. "CurrencyExchange" is pretty bad name for function. Would it make more sense to call it something like "getCurrencyRate" ? Normally names of functions should start from verb.